src/clj/fluree/db/query/sparql/translator.cljc

```diff
@@ -1,7 +1,8 @@
 (ns fluree.db.query.sparql.translator
-  (:require [fluree.db.constants :as const]
+  (:require #?(:cljs [cljs.tools.reader :refer [read-string]])
             [clojure.string :as str]
-            #?(:cljs [cljs.tools.reader :refer [read-string]])))
+            [fluree.db.constants :as const]
+            [fluree.db.util.log :as log]))
```

Comment:
It doesn't look like we're using this `log` namespace.

Reply:
Fixed